beta-src/src/components/WDHomeIcon.tsx

```diff
@@ -0,0 +1,18 @@
+import * as React from "react";
+import { SvgIcon, Box } from "@mui/material";
+import HomeIcon from "./svgr-components/HomeIcon";
+
+const WDHomeIcon: React.FC = function () {
+  return (
+    <Box component="a" href="https://webdiplomacy.net/">
```

Comment:
why is this a link? i think it's supposed to present us with a confirmation dialog box first in some cases, right?